tap_hubspot/__init__.py

```diff
@@ -159,17 +162,23 @@ def get_field_schema(field_type, extras=False):
             }
         }
 
-def parse_custom_schema(entity_name, data):
+def parse_custom_schema(entity_name, data, force_extras=None):
+    if force_extras is not None:
+        extras = force_extras
+    else:
+        extras = entity_name != 'contacts'
     return {
```

Comment:
Nitpick, but can you add a line break before the return here?

Reply:
I'm leaving this since it's the first line now

Reply:
Sure